presto-main/src/main/java/com/facebook/presto/util/MergeSortedPages.java

```diff
@@ -153,14 +154,14 @@ private MergeSortedPages() {}
                 int position;
 
                 @Override
-                public ProcessorState<PageWithPosition> process()
+                public ProcessState<PageWithPosition> process()
                 {
                     if (position >= page.getPositionCount()) {
                         memoryContext.close();
-                        return ProcessorState.finished();
+                        return finished();
                     }
 
-                    return ProcessorState.ofResult(new PageWithPosition(page, position++));
+                    return ofResult(new PageWithPosition(page, position++));
```

Comment:
hm. I don't see static import for `ofResult`

Reply:
https://github.com/prestodb/presto/pull/11778#discussion_r227962965